packages/utils/src/eslint-utils/parserPathSeemsToBeTSESLint.ts

```diff
@@ -0,0 +1,3 @@
+export function parserPathSeemsToBeTSESLint(parserPath: string): boolean {
```

Comment:
[Bug] Hmm, I'm remembering now that `@typescript-eslint/parser` is sometimes wrapped by other parser:

* https://github.com/ota-meshi/eslint-plugin-astro/tree/417e6ae85d925f8b36a56a9d04b0db6a966cb5cc#configuration
* https://eslint.vuejs.org/user-guide/#how-to-use-a-custom-parser

So, perhaps checking based on the `parserPath` is not a feasible approach...

Reply:
Ok, I think I've got a compromise: 

* Here: augmenting the existing error message instead of adding a new reason to throw
* Separating discussion for typed rules out in #8150